backend/src/main/kotlin/blue/mild/covid/vaxx/monitoring/JsonLoggingLayout.kt

```diff
@@ -26,8 +26,8 @@ class JsonLoggingLayout : LayoutBase<ILoggingEvent>() {
         val finalMap: MutableMap<String, Any> = mutableMapOf(
             "@timestamp" to formatTime(event),
             "message" to event.formattedMessage,
-            "logger" to event.loggerName.takeLastWhile { it != '.' },
-            "level" to event.level.levelStr
+            "logger" to event.loggerName.takeLastWhile { it != '.' }, // take only names without packages
```

Comment:
Neni to skoda, ze neuvidime cele jmeno?

Reply:
hele ja bych si myslel, ze je to jedno, alespon je ten log hezci, prijde mi lepsi filtrovat pres
```
docker logs be | jq 'select(.logger == "HttpCallLogger")'
```
nez pres 
```
docker logs be | jq 'select(.logger == "blue.mild.covid.vaxx.HttpCallLogger")'
```

moc pismen